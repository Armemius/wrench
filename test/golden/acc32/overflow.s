    .data
max_pos:         .word  0x7fff_ffff
const_1:         .word  1
const_neg_1:     .word  -1


    .text

_start:
    ;; should not be simulated due to memory errors, but should compile with cropped values
    load         max_pos
    add          const_1
    bvs          clean_overflow
    halt                                     ;; should be skipped
clean_overflow:
    clv

    add          const_neg_1
    halt
