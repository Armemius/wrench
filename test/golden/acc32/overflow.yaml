name: error symbols output
limit: 400
memory_size: 0x1000
reports:
  - name: Step-by-step
    slice: all
    view: |
      {pc}:	{instruction}	{pc:label} Acc: {Acc} Overflow: {V} Carry: {C}
